locale/fi/LC_MESSAGES/django.po

```diff
@@ -2523,7 +2523,7 @@ msgstr "Lataa"
 msgid ""
 "\n"
 "    %(limit)s latest submissions.\n"
-"    <a class=\"btn btn-default btn-sm\" href=\"%(url)s\">Show all</a>\n"
+"    <a class=\"aplus-button--secondary aplus-button--sm\" href=\"%(url)s\">Show all</a>\n"
```

Comment:
You must replicate this change in the Finnish translation below.

Reply:
Thanks, missed that one!

Reply:
It looks like you still didn't commit the change in the django.po file. This is important to fix.